packages/jest-editor-support/src/Runner.js

```diff
@@ -89,6 +92,12 @@ module.exports = class Runner extends EventEmitter {
   }
 
   closeProcess() {
-    this.debugprocess.kill();
+    if (process.platform === 'win32') {
+      // Windows doesn't exit the process when it should.
+      spawn('taskkill', ['/pid', '' + this.debugprocess.pid, '/T', '/F']);
```

Comment:
.kill also is async in that it doesn't guarantee immediate termination of all process.  So I think it's ok that this is also async.